src/__tests__/ShallowWrapper-spec.js

```diff
@@ -484,6 +484,53 @@ describe('shallow', () => {
       expect(spy.args[3][0].hasClass('bux')).to.equal(true);
     });
 
+    describeIf(!REACT013, 'stateless functional components', () => {
+      it('finds nodes', () => {
```

Comment:
these tests don't really seem to correspond with the behavior that was changed in this PR.... right?